src/components/general/AlertContainer.js

```diff
@@ -29,8 +29,9 @@ const AlertContainer = ({
   return (
     <div className="alert-container" style={{marginRight: '10px'}}>
       {alertMessage && <Alert
-        bsStyle={alertStyle}
-        onDismiss={dismissAlert}>
+        variant={alertStyle}
+        onClose={() => dismissAlert()}
```

Comment:
This is small, but unless we're aiming to get more extensibility out of doing

```
onClose={() => {
  // Can add more shit here
  dismissAlert();
  // ...or here
}
```

then we'd might as well turn lines like

```js
onClose={() => dismissAlert()}
```

into

```js
onClose={dismissAlert}
```

Yeah? Slightly more efficient by creating one less function?

Reply:
Yep, I can update this. I forget why I ended up at an anonymous arrow function here anyway.

Reply:
:+1: